### ISSUE TITLE
"No map has been loaded" at addSource()

### ISSUE THREAD

**Issue by ignacio68** [nativescript-community/ui-mapbox#14]
Hello,

I can't add a Geoson layer with addSource(), always return "No map has been loaded" message. With addLayer() doesn't work either because it calls addSource(). 

I've read the same question at @Yermo repository:
Yermo/nativescript-mapbox#356
Yermo/nativescript-mapbox#367
but I can't solve this issue.

My dependences are:
 ```
"dependencies": {
    "@nativescript-community/ui-mapbox": "^6.0.7",
    "@nativescript/core": "~7.0.0",
    "@nativescript/theme": "~2.3.0",
    "@vue/devtools": "^5.3.3",
    "nativescript-socketio": "^3.3.1",
    "nativescript-toasty": "^3.0.0-alpha.2",
    "nativescript-vue": "~2.8.0",
    "nativescript-vue-devtools": "^1.4.0"
  },
 ```
Thanks in advanced

**Comment by cvietor** [comment]
that is fixed and will be part of the next release, which i hope will be in the next days. i just want to finish one more fix

**Comment by ignacio68** [comment]
Ok, thanks four your quick answer. Yes, I've seen the code fixed at the repository.
I will be attentive to the new release.

**Comment by cvietor** [comment]
should be resolved with `6.1.0`
